renovate.json

```diff
@@ -1,12 +1,24 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "automerge": false,
-  "dependencyDashboard": true,
+  "extends": [
+    "config:recommended",
+    ":dependencyDashboard",
+    "schedule:weekly",
+    ":prHourlyLimitNone",
+    ":prConcurrentLimitNone",
+    ":semanticPrefixFixDepsChoreOthers",
+    "group:monorepos",
+    "group:recommended",
+    "group:allNonMajor",
```

Comment:
Group all monorepos, recommended, and all non-major.

- https://docs.renovatebot.com/presets-group/#groupmonorepos
- https://docs.renovatebot.com/presets-group/#grouprecommended
- https://docs.renovatebot.com/presets-group/#groupallnonmajor